categories_map.py

```diff
@@ -1711,6 +1711,15 @@
 #explicit `type label`. Only valid dpg/pog groups can be added as labels
 #Format: label = [cmssw_pkg1, cmssw_pkgN]
 #where cmssw_pkgX can be full or part of start of cmssw Package name
-# "jetmet" = [ "SubSystem1", "SubSystem2/Package"]
+# "jetmet": [ "SubSystem1", "SubSystem2/Package"]
 CMSSW_LABELS = {
+  "tracking": [
+    "CUDADataFormats/Track",
+    "DataFormats/Track",
+    "RecoPixelVertexing",
+    "RecoTracker",
+    "RecoVertex",
+    "TrackPropagation",
+    "TrackingTools"
```

Comment:
Not sure if we need also `RecoPixelVZero` and `RecoVZero` (do not exist anymore since a long time).
Do we need to flag also non reconstruction packages?

Reply:
> Do we need to flag also non reconstruction packages?

good point, here is more that matches

- CUDADataFormats/Vertex 
- DataFormats/VertexReco
- DQM/Tracking
- Validation/RecoPixelVertexing
- Validation/RecoTrack
- Validation/RecoVertex
- Validation/Tracking
- FastSimulation/Tracking
- Geometry/GlobalTracking 

what else?

Reply:
> Not sure if we need also `RecoPixelVZero` and `RecoVZero` (do not exist anymore since a long time).

these were apparently removed in 9_2_X ( cms-sw/cmssw#18623) 10_0_X (cms-sw/cmssw#21456), respectively
But I guess I can add these as well

Reply:
I would add:
   * CondFormats/BeamSpotObjects
   * AnalysisDataFormats/TrackInfo 
   * CondCore/BeamSpotPlugins
   * CondFormats/BeamSpotObjects
   * CondTools/BeamSpot
   * DQM/BeamMonitor
   * Validation/TrackingMCTruth

Reply:
> I would add:

now in f2c36cc

Reply:
actually in 9d8c766